src/resolvers.ts

```diff
@@ -0,0 +1,269 @@
+import {Prisma} from '@prisma/client'
+// may not need to import prisma - but idk how to make sure input is valid
+
+//_parent, _args, context: Context
+// Use this if context includes prismaclient
+export const resolvers = {
+  Query: {
+    users: async (context) => {
```

Comment:
It would be good to refactor these resolvers into separate files based on which type they resolve.